### DISCUSSION TITLE
How to load only particular countries map in online mode.

### DISCUSSION THREAD

**Discussion by endlesjoy** [osmdroid/osmdroid#1927] [Q&A]
Hello everybody, can you tell, how can i load only map of some country? I can't find anything in internet. Is this feature available?

**Comment by spyhunter99** [comment]
osmdroid only displays map tiles provided by some tile source. If you can find a tile source that has just what you're looking for, that might work. The mapsforge adapter might help you out too. They have data sets for specific countries
